makefile

```diff
@@ -53,19 +55,50 @@ $(error Building with the parameters library is not available on Windows OS)
 endif
 endif
 
-COMPILERs = icc icx gnu clang vc
-COMPILER ?= icc
+ifeq ($(PLAT),lnx32e)
+  BACKEND_CONFIG ?= mkl
+  ARCH = 32e
+else ifeq ($(PLAT),mac32e)
+  BACKEND_CONFIG ?= mkl
+  ARCH = 32e
+else ifeq ($(PLAT),lnxarm)
+  BACKEND_CONFIG ?= ref
+  ARCH = arm
+else
+  $(error Unsupported platform: $(PLAT))
```

Comment:
for call make oneapi BUILDREV= "CHECK_DLL_SIG=yes"  -j24 REQCPU=avx2 
on windows platform. 
makefile:68: *** Unsupported platform: win32e.  Stop.

This have to be fixed to allow windows build as well . not sure what happening in public CI though